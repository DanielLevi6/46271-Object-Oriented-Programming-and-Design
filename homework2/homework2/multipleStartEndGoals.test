# multipleStartEndGoals test file

# Create a graph
CreateGraph myGraph

# Add nodes
CreateNode node1 1
CreateNode node2 2
CreateNode node3 3
CreateNode node4 4
CreateNode node5 5
CreateNode node6 6
CreateNode node7 7
CreateNode node8 8

AddNode myGraph node1
AddNode myGraph node2
AddNode myGraph node3
AddNode myGraph node4
AddNode myGraph node5
AddNode myGraph node6
AddNode myGraph node7
AddNode myGraph node8

# Add an edge
AddEdge myGraph node1 node2
AddEdge myGraph node1 node3
AddEdge myGraph node1 node4
AddEdge myGraph node2 node3
AddEdge myGraph node3 node4
AddEdge myGraph node2 node5
AddEdge myGraph node1 node5
AddEdge myGraph node1 node6
AddEdge myGraph node5 node8

# Find a path
FindPath myGraph node1 node2 node3 -> node6 node7 node8