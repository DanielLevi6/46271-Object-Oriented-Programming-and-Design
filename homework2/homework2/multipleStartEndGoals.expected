# multipleStartEndGoals test file

# Create a graph
created graph myGraph

# Add nodes
created node node1 with cost 1
created node node2 with cost 2
created node node3 with cost 3
created node node4 with cost 4
created node node5 with cost 5
created node node6 with cost 6
created node node7 with cost 7
created node node8 with cost 8

added node node1 to myGraph
added node node2 to myGraph
added node node3 to myGraph
added node node4 to myGraph
added node node5 to myGraph
added node node6 to myGraph
added node node7 to myGraph
added node node8 to myGraph

# Add an edge
added edge from node1 to node2 in myGraph
added edge from node1 to node3 in myGraph
added edge from node1 to node4 in myGraph
added edge from node2 to node3 in myGraph
added edge from node3 to node4 in myGraph
added edge from node2 to node5 in myGraph
added edge from node1 to node5 in myGraph
added edge from node1 to node6 in myGraph
added edge from node5 to node8 in myGraph

# Find a path
shortest path in myGraph: node1 node6
